NOTICE.md

```diff
@@ -64,4 +64,10 @@ they are given here (manually added) for time being:
 |paho-mqtt-cpp|1.2.0|EPL 1.0 AND EDL 1.0|
 |protobuf|3.21.4|Google License|
 |re2|20220601|BSD-3-Clause|
-|zlib|1.2.12|zlib/libpng license|
\ No newline at end of file
+|zlib|1.2.12|zlib/libpng license|
+
+### Additional 3rd-party dependencies
+
+| Dependency | Source | License |
+|:-----------|:-------:|--------:|
+|.devcontainer/scripts/reinstall-cmake.sh|https://github.com/microsoft/vscode-dev-containers/tree/main/containers/cpp/.devcontainer|MIT|
```

Comment:
Do we really need to list it separately? I feel the script is already attributed since it comes from MS's cpp dev container which is already listed above.

Reply:
Marcel Kurzmann asked us to do it because it was found in a scan...

Reply:
I put it now to the devcontainer section, but mentioned it explicitly... Hope that's also fine for you?